templates/contactme.html

```diff
@@ -1,4 +1,52 @@
-{% extends "base.html" %}
-{% block content %}
+{% extends "base.html" %} 
+
+<!--extend the functionality from the header of the parent component.-->
+{% block head %}
+    {{ super() }}
```

Comment:
why does this blockhead call super but base does not?

Reply:
using super() in the child class allows me to extend the functionality of the parent class "(base.html)".